src/FlagProvider.tsx

```diff
@@ -41,6 +41,11 @@ const FlagProvider: React.FC<IFlagProvider> = ({
 
   React.useEffect(() => {
     client.current.start();
+
+    return () => {
```

Comment:
Add client.current.stop here too?

Reply:
Makes sense. Could move the `client.current.on` calls into `useEffect` as well.